website/project/utils.py

```diff
@@ -118,3 +118,43 @@ def sizeof_fmt(num, suffix='B'):
             return '%3.1f%s%s' % (num, unit, suffix)
         num /= 1000.0
     return '%.1f%s%s' % (num, 'Y', suffix)
+
+
+def get_storage_limits_css(node):
+    status = node.storage_limit_status
+    GBs = 10 ** 9
+    public_limit = node.custom_storage_usage_limit_public or settings.STORAGE_LIMIT_PUBLIC
+    private_limit = node.custom_storage_usage_limit_private or settings.STORAGE_LIMIT_PRIVATE
+
+    over_public_size = sizeof_fmt(int(public_limit * GBs))
+    over_private_size = sizeof_fmt(int(private_limit * GBs))
+
+    if not node.is_public:
+        helpful_message = f'Make this project public to extend the storage limit to {over_public_size}.'
+    else:
+        helpful_message = ''
+
+    if status is settings.StorageLimits.APPROACHING_PRIVATE:
+        return {
+            'class': 'btn-warning storage-warning',
+            'text': f'This node is approaching its limit of {over_private_size}. ' + helpful_message
+        }
+    elif status is settings.StorageLimits.OVER_PRIVATE:
+        return {
+            'class': 'btn-danger  storage-warning',
+            'text': f'This node has exceeded its limit of {over_private_size}. ' + helpful_message
+        }
+    elif status is settings.StorageLimits.APPROACHING_PUBLIC:
+        return {
+            'class': 'btn-warning  storage-warning',
+            'text': f'This node is approaching its limit of {over_public_size}.'
+        }
+    elif status is settings.StorageLimits.OVER_PUBLIC:
+        return {
+            'class': 'btn-danger  storage-warning',
+            'text': f'This node has exceeded its limit of {over_public_size}.'
+        }
```

Comment:
Here is the [updated language](https://docs.google.com/document/d/13dZBXO_JqZOcf4fe-HKCk_YDMA5PYokfOOw4KB3jbEY/edit?ts=5f47d7b8#bookmark=id.ga5sxbogg83u) for these and the error notification. Check with Kayla to see if she has a link to the help docs that we can use.